profile/README.md

```
@@ -1,9 +1,9 @@
 <p align="center">
 <a href="https://www.traceloop.com/#gh-light-mode-only">
-<img width="300" src="https://raw.githubusercontent.com/traceloop/.github/main/img/traceloop-light.png">
+<img width="300" src="https://raw.githubusercontent.com/traceloop/.github/main/img/traceloop-by-servicenow-light.png">
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)